compiler/rustc_ast_pretty/src/pprust/state.rs

```diff
@@ -679,6 +681,7 @@ pub trait PrintState<'a>: std::ops::Deref<Target = pp::Printer> + std::ops::Dere
                     false,
                     None,
                     *delim,
+                    spacing.open,
```

Comment:
Changes in `use a::{b, c}` and `Foo<{true}>` are caused by this line, and are related to token stream printing, not to AST printing.
So they do not need adding the `open_spacing: Spacing` field to AST, token streams already have the relevant information.

I guess for AST it would also be good enough to just use `Spacing::Alone` for `{` and `Spacing::Joint` for `(`/`[` instead of adding the field.

Reply:
Hmm, good observation. I'll remove the `DelimArgs` changes.